### DISCUSSION TITLE
Prioritize core EVENT features

### DISCUSSION THREAD

**Discussion by islandmix** [HiEventsDev/Hi.Events#288] [General]
Now that the "ticket as product enhancement" is done... How do we prioritize the following core event features on the roadmap... 

- Support Android/Apple wallets
- Address field based on location
- Show user currency (nice to have)

Everything else in the current pipeline have good enhancement suggestions but 2 of the above are core basic requirements (IMHO) before this can be used in a prod environment. I am also seeing a lot of requests that try to incorporate every single type of event scenario: e.g.) booking classes, seat maps etc... but..... respectfully...  can we get the core features first before this tool is everything to everyone doing an event... 

I know that we want to be better than the other platforms but lets get similar core features working first.... Lets also share this project so more developers can help

**Comment by daveearley** [comment]
Thanks for the feedback! Wallet support and location-based address fields are on the roadmap, but for now, we’re focused on features that make Hi.Events more attractive to larger events and improving the architecture for stability and scalability. These updates are critical to building a strong foundation for everyone.

If there’s a specific feature you need sooner, I’d encourage contributing directly—unfortunately, with open-source development, certain things can take time unless the community pitches in. We’d love to see more contributors jump in and help shape the project! 😊
